config.yml

```diff
@@ -147,6 +147,10 @@
 #elasticsearch.bulk.retry_interval: 10
 #
 #
+##  Enable to log ids of created/indexed/deleted/updated documents during a sync.
```

Comment:
help me make sure that this isn't lost with https://github.com/elastic/connectors/pull/2280 (depending on which merges first)

Reply:
mine merged first. To resolve the conflict, this'll just need to be moved to `config.yml.example`